src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -44,7 +44,7 @@ public SearchIndexFromCatalogCollector(ILogger logger, Uri index, IndexWriter in
             IEnumerable<JObject> catalogItems = await FetchCatalogItems(client, items, cancellationToken);
 
             var numDocs = _indexWriter.NumDocs();
-            _logger.LogInformation(string.Format("Index contains {0} documents.", _indexWriter.NumDocs()));
+            _logger.LogInformation(string.Format("Index contains {0} documents.", numDocs));
```

Comment:
I think this is supposed to be

`_logger.LogInformation($"Index contains {NumDocs} documents.", numDocs);`

to follow Serilog format, but I don't think it's that big of a deal.